src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -52,567 +51,718 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
+                 c.FirstName != oldContacts[i].FirstName ||
+                 c.Exclude_from_Household_Name__c != oldContacts[i].Exclude_from_Household_Name__c) &&
+                c.AccountId != NULL &&
+                checkHouseholdNaming(c.Account.RecordTypeId))
+            {
+                accountIdToRename.add(c.AccountId);
             }
         }
+    }
 
-        // now done going thru all contacts, do our DML logic based on our lists
+    /*******************************************************************************************************
+    * @description This method checks if any of the after update collections are not empty.
+    * If not empty, it calls the appropriate methods to process the list/set/map.
+    * param updateWrapper contains the collections processed from after update context.
+    * param contactsNeedAccounts a list of Contacts that needs new Accounts or an update.
+    * @return void.
+    */
+    private void handleUpdateProcessing(HandleUpdateWrapperLogic updateWrapper, Contact[] contactsNeedAccounts) {
+        DmlWrapper dmlWrapper = new DmlWrapper();
 
-        if (listContactNeedAccount.size() > 0) {
-        	UTIL_Debug.debug('****Number of contacts that need accounts created: ' + listContactNeedAccount.size());
-            //add the newly created or updated Contacts that need a new individual account
-            insertContactAccount(listContactNeedAccount, dmlWrapper);
-        }
-        if (mapAccountIdContactId.size() > 0) {
-            //update Accounts that have newly created Contacts connected to them
-            updateAccountPrimaryContact(mapAccountIdContactId, dmlWrapper);
-            //we also need to create affiliations for those contacts that have been created as children of existing accounts,
-            //and we have to populate the key affiliation field on those contacts
-            //createAfflsToExistingAccs(mapAccountIdContactId, dmlWrapper);
+        //Creates new Account
+        if (contactsNeedAccounts.size() > 0) {
+            UTIL_Debug.debug('****Number of Contacts that need Accounts created: ' + contactsNeedAccounts.size());
+            insertContactAccount(contactsNeedAccounts, dmlWrapper);
         }
 
-        if (listAccountIdHHToUpdate.size() > 0) {
-            updateParentAcc(listAccountIdHHToUpdate, dmlWrapper);
-        }
-        if (mapContactIdContactOwnerChange.size() > 0) {
-            updateOwners(mapContactIdContactOwnerChange, dmlWrapper);
-        }
-        if (listContactAccountDelete.size() > 0) {
-        	UTIL_Debug.debug('****Number of accs to delete: ' + listContactAccountDelete.size());
-            // check all old accounts to delete is they are left hanging around
-            deleteContactAccountsIfEmpty(listContactAccountDelete, dmlWrapper);
+        if (triggerAction == TDTM_Runnable.Action.AfterUpdate) {
+            List<Contact> contactChangedLastName = updateWrapper.contactsChangedLastName;
```

Comment:
Same comments that I added to the other processing method - do we need these local variables? Can we just update the logic to pass in the properties in the updateWrapper?

For example, line 349: `if (updateWrapper.accountIdToRename.size() > 0) {`